test/integration/offline-custom-assets.js

```diff
@@ -52,22 +51,16 @@ test('offline-custom-assets', t => {
             t.equals(costumes.length, 1);
             const customCostume = costumes[0];
             t.equals(customCostume.name, 'A_Test_Costume');
-            const costumeMd5Ext = customCostume.md5;
-            const costumeIdParts = StringUtil.splitFirst(costumeMd5Ext, '.');
-            const costumeMd5 = costumeIdParts[0];
 
-            const storedCostume = vm.runtime.storage.get(costumeMd5);
+            const storedCostume = customCostume.asset;
```

Comment:
It's great that we can now replace all those lines of setup of trying to get the data out of storage with just this one line lookup. =D